p2p/transport/webrtc/sdp.go

```diff
@@ -0,0 +1,90 @@
+package libp2pwebrtc
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/multiformats/go-multihash"
+)
+
+type sdpArgs struct {
+	Addr        *net.UDPAddr
+	Ufrag       string
+	Password    string
+	Fingerprint *multihash.DecodedMultihash
+}
+
+const CLIENT_SDP string = `
```

Comment:
1. This probably doesn't need to be exported. `clientSDP` would be the more appropriate name.
2. Should this use https://pkg.go.dev/text/template#Template? That would certainly be more expressive, but might come with a performance penalty.